src/core_plugins/timelion/public/directives/timelion_expression_input_helpers.js

```diff
@@ -73,6 +73,26 @@ function inLocation(cursorPosition, location) {
   return cursorPosition >= location.min && cursorPosition <= location.max;
 }
 
+function getArgumentsHelp(functionHelp, functionArgs = []) {
+  if (!functionHelp) {
+    return [];
+  }
+
+  // Do not provide 'inputSeries' as argument suggestion for chainable functions
+  const argsHelp = functionHelp.chainable ? functionHelp.args.slice(1) : functionHelp.args.slice(0);
+
+  // ignore arguments that are all ready provided in function declaration
+  const providedArguments = functionArgs.map((arg) => {
```

Comment:
Minor suggestion: when I'm mapping out properties, I like to name the variable with the original array + the property name e.g. this would become `functionArgNames`. This helps me tie the variable back to the original one and lets me know what it contains.